src/mutmap/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Rscript)
+
+set(MUTMAP_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/mutmap)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mutmap.sh.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/mutmap.sh" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/build.R"
+  "${CMAKE_CURRENT_BINARY_DIR}/build.R" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/bundle.js"
+  "${CMAKE_CURRENT_BINARY_DIR}/bundle.js" @ONLY)
+
```

Comment:
You would be right if we were designing a web app. However, we are not designing a web app. We are designing an interactive report. The code does not need to be modular since we are unlikely to reuse it in other parts of DeNovoGear anytime soon. We also are already using a building/packaging system in CMake and should use that as much as we can to bundle source files together into the template system. Written correctly, it is as simple as "cat *.js > output.js".

We also likely don't need typescript support. Depending on node might make the developer's life easier, but adds an additional dependency to the use of DeNovoGear. And one that doesn't have anything to do with runtime benefits.